diff --git a/pos_next/api/__init__.py b/pos_next/api/__init__.py
index 51ebb2a8..1d00f62d 100644
--- a/pos_next/api/__init__.py
+++ b/pos_next/api/__init__.py
@@ -12,7 +12,7 @@ from . import customers
 from . import offers
 from . import promotions
 from . import utilities
-from . import auth
+from . import customer_display
 
 @frappe.whitelist(allow_guest=True)
 def ping():
diff --git a/pos_next/hooks.py b/pos_next/hooks.py
index 84ba7867..753ae2db 100644
--- a/pos_next/hooks.py
+++ b/pos_next/hooks.py
@@ -87,10 +87,40 @@ _asset_version = get_build_version()
 # Fixtures
 # --------
 fixtures = [
+	{
+		"dt": "Custom Field",
+		"filters": [
+			[
+				"name",
+				"in",
+				[
+					"Sales Invoice-posa_pos_opening_shift",
+					"Sales Invoice-posa_is_printed",
+					"Item-custom_company",
+					"POS Profile-posa_cash_mode_of_payment",
+					"POS Profile-posa_allow_delete",
+					"POS Profile-posa_block_sale_beyond_available_qty",
+					"Mode of Payment-is_wallet_payment"
+				]
+			]
+		]
+	},
+	{
+		"dt": "Print Format",
+		"filters": [
+			[
+				"name",
+				"in",
+				[
+					"POS Next Receipt"
+				]
+			]
+		]
+	},
     {
         "dt": "Role",
         "filters": [
-            ["role_name", "in", ["POSNext Cashier","Nexus POS Manager"]]
+            ["role_name", "in", ["POSNext Cashier"]]
         ]
     },
     {
@@ -172,13 +202,7 @@ doc_events = {
 		"validate": "pos_next.validations.validate_item"
 	},
 	"Customer": {
-		"after_insert": [
-			"pos_next.api.customers.auto_assign_loyalty_program",
-			"pos_next.realtime_events.emit_customer_event",
-			"pos_next.api.wallet.create_wallet_on_customer_insert"
-		],
-		"on_update": "pos_next.realtime_events.emit_customer_event",
-		"on_trash": "pos_next.realtime_events.emit_customer_event"
+		"after_insert": "pos_next.api.customers.auto_assign_loyalty_program"
 	},
 	"Sales Invoice": {
 		"validate": [
@@ -195,9 +219,6 @@ doc_events = {
 	},
 	"POS Profile": {
 		"on_update": "pos_next.realtime_events.emit_pos_profile_updated_event"
-	},
-	"Promotional Scheme": {
-		"on_update": "pos_next.overrides.pricing_rule.sync_pos_only_to_pricing_rules"
 	}
 }
 
